File: API/Entities/Station.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace API.Entities
{
    public class Station
    {
        public int Id { get; set; }
        public string StationName { get; set; }
        public double GegrLat { get; set; }
        public double GegrLon { get; set; }
        public int CityId { get; set; }
        public virtual City City { get; set; }
        public string AddressStreet { get; set; }
    }
}

File: client/src/app/models/station.ts
import { Sensor } from "./sensor";
import { StationState } from "./stationState";

export interface Station{
    id: number,
    stationName: string,
    gegrLat: number,
    gegrLon: number,
    addressStreat: string,
    sensors?: Sensor[],
    stationState?: StationState
}